packages/@ourworldindata/grapher/src/core/LegacyToOwidTable.ts

```diff
@@ -551,13 +551,12 @@ const variableTypeToColumnType = (type: OwidVariableType): ColumnTypeNames => {
     switch (type) {
         case "ordinal":
             return ColumnTypeNames.Ordinal
-        // TODO
-        // case "string":
-        //     return ColumnTypeNames.String
-        // case "float":
-        // case "int":
-        //     return ColumnTypeNames.Numeric
-        // case "mixed":
+        case "string":
+            return ColumnTypeNames.String
+        case "float":
+        case "int":
```

Comment:
We already have `ColumnTypeNames.Integer`, which I think will be a good fit for `"int"`.

The main difference to `Numeric` is that it formats numbers with 0 decimal places by default.

Reply:
Sounds good! Do you think we should use `NumericColumn` for float types directly, or should we add another `FloatColumn` that inherits from `NumericColumn`?

Reply:
I think it's fine to just use `NumericColumn`.

It's not like a `float` type is _only_ floats - there'll be integer values in these columns, too.

Reply:
Makes sense!